### DISCUSSION TITLE
python demos.. wifi_connect

### DISCUSSION THREAD

**Discussion by dbstpfk** [gopro/OpenGoPro#99] [Q&A] [answered]
I ran the gopro "demos->python->tutorial->tutorial modules->tutorial 5_connect_wifi->tutorial 5_connect_wifi -> wifi_enable_and_connect.py " file.
 but i get this error " ModuleNotFoundError: No module named 'open_gopro.wifi_controller' " 
What is the solution? so....sad... ㅠㅠ

**Comment by tcamise-gpsw** [comment]
Hello. I believe this is the same problem as https://github.com/gopro/OpenGoPro/issues/103

I am looking into this and will update the issue as it is solved.

**Comment by tcamise-gpsw** [comment, marked as answer]
This should be fixed now via #109.  I hard-coded the OpenGoPro version to prevent this from happening again as well as added a test suite.
